www/include/configuration/configObject/service/listServiceByHostGroup.php

```diff
@@ -89,16 +89,28 @@
 $rows = 0;
 $tmp = null;
 $tmp2 = null;
-$searchHG = $pearDB->escape($searchHG);
-$searchS = $pearDB->escape($searchS);
+$searchHG = \HtmlAnalyzer::sanitizeAndRemoveTags(
```

Comment:
if i remove it the bug of yesterday will appear again :( either i let the escape or initialize the variable with something otherwise it will say "undefined variable".